auth_backends/urls.py

```diff
@@ -0,0 +1,12 @@
+""" Shared URL patterns.
+
+Add these to your project's `urlpatterns` to avoid duplicating code.
+"""
+from django.conf.urls import url, include
+
+from auth_backends.views import EdxOpenIdConnectLoginView
+
+auth_urlpatterns = [  # pylint: disable=invalid-name
+    url(r'^login/$', EdxOpenIdConnectLoginView.as_view(), name='login'),
```

Comment:
I take it that someone wanting to use this is going to have to include their subclassed version of `LogoutRedirectBaseView` here? Would a comment to that effect make sense here?